Slim/Utils/Prefs.pm

```diff
@@ -268,15 +268,13 @@ sub init {
 		# Bug 5557, disable UPnP support by default
 		'noupnp'                => 1,
 		'onlyAlbumYears'        => 1,
+		'artistAlbumLink'       => 1,
+		'albumartistAlbumLink'  => 1,
+		'trackartistAlbumLink'  => $prefs->get('useUnifiedArtistsList') && $prefs->get('trackartistInArtists'),
+		'composerAlbumLink'     => $prefs->get('useUnifiedArtistsList') && $prefs->get('composerInArtists'),
+		'conductorAlbumLink'    => $prefs->get('useUnifiedArtistsList') && $prefs->get('conductorInArtists'),
+		'bandAlbumLink'         => $prefs->get('useUnifiedArtistsList') && $prefs->get('bandInArtists'),
```

Comment:
Just set them to the same value as the corresponding `...InArtists` pref. On a new system you can't expect those prefs to be defined in order. It's a hash, not a list. The order in which the items are processed is not defined.

Reply:
That's what it's doing, except only if the user is using the combined list. This keeps things the same as they are now for separate list users, where they only ever get ALBUMARTIST/ARTIST album lists (unless they're browsing from a role or roles).

I'm not attempting to do anything about sequencing - they'll be alphabetic by role name when in use, as per the sort in Contributor.pm.

Reply:
Fair enough. It probably doesn't really matter.